content/post/20211010-introducing-coalton.md

```diff
@@ -291,10 +299,12 @@ The Lisp-calls-Coalton bridge is just as easy with the `coalton` operator. For i
 55
 
 > (coalton (fib 10.0))
-; Failed to unify types SINGLE-FLOAT and INTEGER
-; in unification of types (SINGLE-FLOAT → :A) and (INTEGER → INTEGER)
-; in COALTON
-;    [Condition of type COALTON-IMPL/TYPECHECKER::COALTON-TYPE-ERROR-CONTEXT]
+;error: Type mismatch
+;  --> COALTON (NIL):1:14
```

Comment:
was the reader on when you did this?

Reply:
I had forgotten to recompile (named-readtables:in-readtable coalton:coalton), now I get:
```
error: Type mismatch
  --> <unknown>:1:14
   |
 1 |  (coalton (fib 1.0))
   |                ^^^ Expected type 'INTEGER' but got type 'DOUBLE-FLOAT'
   [Condition of type COALTON-IMPL/TYPECHECKER/BASE:TC-ERROR]
```

Is this preferable?

Reply:
yeah that's fine